src/views/CountryThreeWProjects/index.tsx

```diff
@@ -0,0 +1,515 @@
+                        />
+                    </div>
+                    <div className={styles.keyFigures}>
+                        <KeyFigure
+                            className={styles.keyFigure}
+                            value={ongoingProjectBudget}
+                            description={strings.ongoingProjectBudgetTitle}
+                        />
+                        <div className={styles.separator} />
+                        <PieChart
+                            data={projectStatusTypeStats}
+                            valueSelector={numericValueSelector}
+                            labelSelector={stringLabelSelector}
+                            keySelector={stringLabelSelector}
+                            colors={primaryRedColorShades}
+                        />
+                    </div>
+                </div>
+            )}
+            <Container
+                heading={strings.threeWOngoingProjectsTitle}
+                withHeaderBorder
+                childrenContainerClassName={styles.content}
+                filters={(
+                    <Filter
+                        value={filters}
+                        onChange={setFilters}
+                        districtList={districtList}
+                    />
+                )}
+                actions={(
+                    <>
+                        <Button
+                            variant="primary"
+                            name={undefined}
+                            onClick={handleExportClick}
+                            icons={(<DownloadFillIcon />)}
+                        >
+                            {strings.exportProjects}
+                        </Button>
+                        <Link
+                            to={`/three-w/all/?country=${countryResponse?.iso}`}
+                            withForwardIcon
+                        >
+                            {strings.viewAllProjects}
+                        </Link>
+                    </>
+                )}
+            >
+                <div className={styles.topSection}>
+                    <div className={styles.mapContainer}>
+                        <Map
+                            className={styles.mapContainer}
+                            projectList={ongoingProjects}
+                            districtList={districtList}
+                        />
+                    </div>
+                    <Container
+                        className={styles.sidebar}
+                        heading={strings.threeWInCountryMapSidebarTitle}
+                    >
+                        {Object.entries(districtGroupedProject).map(([districtId, projects]) => {
+                            if (isNotDefined(projects) || projects.length === 0) {
+                                return null;
+                            }
+
+                            const district = districtList.find((d) => d.id === Number(districtId));
```

Comment:
Why do we need to check district?

Reply:
the projects are grouped based on district